src/main/java/uk/gov/hmcts/reform/civil/model/dq/EmploymentTypeComplexTypeDQLRspec.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.civil.model.dq;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+public class EmploymentTypeComplexTypeDQLRspec {
+
+    private final List<String> employmentTypeComplexTypeDQLRspec;
```

Comment:
If I'm not mistaken, this is for front's employed/self-employed checkboxes. I'd use an enum for it, to keep track of what values are possible

Reply:
Removed this class. as complex type was removed from ccd